tools/debug/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+//! Debugging dev tools for Emergence.
+//!
+
+use bevy::{
+    asset::AssetServer,
+    diagnostic::{Diagnostics, FrameTimeDiagnosticsPlugin},
+    prelude::{default, Color, Commands, Component, Query, Res, TextBundle, Transform, With}, // Input, KeyCode, // add back to allow for toggling the fps display
+    text::{Text, Text2dBundle, TextAlignment, TextSection, TextStyle},
+    time::Time,
+    ui::{PositionType, Style, UiRect, Val},
+};
+
+use bevy_ecs_tilemap::map::{TilemapGridSize, TilemapType};
+use bevy_ecs_tilemap::tiles::TilePos;
+// use emergence_lib::graphics::terrain::TerrainTilemap;
```

Comment:
we can just remove this if it is not being used here

Reply:
Doing so in my next commit.